pokemongo_bot/logger.py

```diff
@@ -8,17 +9,25 @@
     lcd = False
 
 def log(string, color = 'white'):
-    colorHex = {
+	# Write bot output also in a log file
+	try:
+		logs_file = open('logs/' + time.strftime("%Y%m%d") + '.txt', 'a')
+		logs_file.write('[' + time.strftime("%H:%M:%S") + '] ' + string + '\n')
+		logs_file.close()
+	except:
```

Comment:
Don't do this. Let exceptions be raised if any error happens on logging to file, please.